clang/test/CodeGenCXX/constructor-attr.cpp

```diff
@@ -10,3 +13,14 @@ struct Foo {
     bar();
   }
 };
+
+template <int P>
```

Comment:
Please write a test where the argument is NTTP dependent on another template argument.